src/vaultController.ts

```diff
@@ -1,4 +1,4 @@
-import type { Hash, Address, PublicClient, WalletClient, Transport, Chain } from "viem";
+import { Hash, Address, PublicClient, WalletClient, Transport, Chain, maxUint256, pad, zeroAddress } from "viem";
```

Comment:
these new imports aren't used anywhere. Also, we should separate `type` imports and normal imports. So bring back the `type` expression for this one.